### PR TITLE
Fixes an intermittently failing swift live test.

### PR DESCRIPTION
**Pull request by zack-shoylev**

### CODE REVIEW

File: apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/internal/BaseSwiftApiLiveTest.java [hunk 1]
```diff
@@ -48,6 +48,10 @@ protected BaseSwiftApiLiveTest() {
       provider = "openstack-swift";
    }
 
+   protected void awaitConsistency() {
+      Uninterruptibles.sleepUninterruptibly(5, TimeUnit.SECONDS);
```
Comment (andrewgaul) [inline apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/internal/BaseSwiftApiLiveTest.java]: Should this use the `test.blobstore.await-consistency-timeout-seconds` property as used by `BaseBlobStoreIntegrationTest`?

**Comment by andrewgaul** [comment]
@zack-shoylev Relevant, S3Proxy now models eventual consistency: https://github.com/andrewgaul/s3proxy/commit/6292d28c9c66e8f432ae5b18acbadd0c191bc05d.  Easy to add this to [SwiftProxy](https://github.com/bouncestorage/swiftproxy).

**Comment by zack-shoylev** [comment]
Thanks @andrewgaul ! I will refactor this a bit, but there are some other swift tests to fix after this, I think. Adding this to swiftproxy is also a good idea - I'll try to look into it some more.

**Comment by zack-shoylev** [comment]
merged

**Pull request closed without merging**